<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AzConvert</string>
  </property>
  <property name="windowIcon">
   <iconset resource="azc_res.qrc">
    <normaloff>:/res/Convert.png</normaloff>:/res/Convert.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <property name="margin">
     <number>5</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTextEdit" name="txtSource">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btnOpen">
             <property name="text">
              <string>&amp;Open...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnPaste">
             <property name="minimumSize">
              <size>
               <width>71</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>&amp;Paste</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextEdit" name="txtResult">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="btnSave">
             <property name="text">
              <string>&amp;Save...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnCopy">
             <property name="text">
              <string>&amp;Copy</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_Convert">
     <property name="tearOffEnabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>&amp;Convert</string>
     </property>
     <addaction name="action_Latin_to_Arabic"/>
     <addaction name="action_Arabic_to_Latin"/>
     <addaction name="action_Cyrillic_to_Latin"/>
    </widget>
    <addaction name="menu_Convert"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <widget class="QMenu" name="menu_Language">
     <property name="title">
      <string>&amp;Language</string>
     </property>
     <addaction name="action_LangEnglish"/>
     <addaction name="action_LangAzerbaijani"/>
     <addaction name="action_LangTurkish"/>
     <addaction name="action_LangAzTurkish"/>
     <addaction name="action_LangPersian"/>
    </widget>
    <addaction name="action_ConvertText"/>
    <addaction name="action_Reload_dictionaries"/>
    <addaction name="action_Add_word_to_dictionary"/>
    <addaction name="separator"/>
    <addaction name="actionCalendar_converter"/>
    <addaction name="separator"/>
    <addaction name="action_Options"/>
    <addaction name="menu_Language"/>
    <addaction name="actionWikipediaMode"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_ConvertText"/>
   <addaction name="separator"/>
   <addaction name="actionClear_All"/>
   <addaction name="separator"/>
   <addaction name="actionCalendar_converter"/>
   <addaction name="action_Add_word_to_dictionary"/>
  </widget>
  <widget class="QToolBar" name="modeToolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Latin_to_Arabic"/>
   <addaction name="action_Arabic_to_Latin"/>
   <addaction name="action_Cyrillic_to_Latin"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Latin_to_Arabic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>L -&gt; A</string>
   </property>
   <property name="toolTip">
    <string>Latin script to Arabic script convert mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="action_Arabic_to_Latin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>A -&gt; L</string>
   </property>
   <property name="toolTip">
    <string>Arabic script to Latin script convert mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="action_Cyrillic_to_Latin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>C -&gt; L</string>
   </property>
   <property name="toolTip">
    <string>Cyrillic script to Latin script convert mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="action_ConvertText">
   <property name="icon">
    <iconset resource="azc_res.qrc">
     <normaloff>:/res/Convert.png</normaloff>:/res/Convert.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Convert</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="action_Reload_dictionaries">
   <property name="text">
    <string>&amp;Reload dictionaries</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_Add_word_to_dictionary">
   <property name="icon">
    <iconset resource="azc_res.qrc">
     <normaloff>:/res/AddWord.png</normaloff>:/res/AddWord.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add word...</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionCalendar_converter">
   <property name="icon">
    <iconset resource="azc_res.qrc">
     <normaloff>:/res/Calendar.png</normaloff>:/res/Calendar.png</iconset>
   </property>
   <property name="text">
    <string>Calen&amp;dar...</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="action_Options">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Options...</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actionClear_All">
   <property name="icon">
    <iconset resource="azc_res.qrc">
     <normaloff>:/res/Clear.png</normaloff>:/res/Clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear All</string>
   </property>
   <property name="toolTip">
    <string>Clear both boxes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
  <action name="action_LangEnglish">
   <property name="icon">
    <iconset resource="azc_res.qrc">
     <normaloff>:/res/Flag_Usa.png</normaloff>:/res/Flag_Usa.png</iconset>
   </property>
   <property name="text">
    <string notr="true">English</string>
   </property>
  </action>
  <action name="action_LangAzerbaijani">
   <property name="icon">
    <iconset resource="azc_res.qrc">
     <normaloff>:/res/Flag_Azerbaijan.png</normaloff>:/res/Flag_Azerbaijan.png</iconset>
   </property>
   <property name="text">
    <string notr="true">Azərbaycanca</string>
   </property>
  </action>
  <action name="action_LangTurkish">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="azc_res.qrc">
     <normaloff>:/res/Flag_Turkey.png</normaloff>:/res/Flag_Turkey.png</iconset>
   </property>
   <property name="text">
    <string notr="true">Türkçe</string>
   </property>
  </action>
  <action name="action_LangAzTurkish">
   <property name="icon">
    <iconset resource="azc_res.qrc">
     <normaloff>:/res/Flag_Iran.png</normaloff>:/res/Flag_Iran.png</iconset>
   </property>
   <property name="text">
    <string notr="true">تورکجه</string>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
    </font>
   </property>
  </action>
  <action name="action_LangPersian">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="azc_res.qrc">
     <normaloff>:/res/Flag_Iran.png</normaloff>:/res/Flag_Iran.png</iconset>
   </property>
   <property name="text">
    <string notr="true">فارسی</string>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
    </font>
   </property>
  </action>
  <action name="actionWikipediaMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Wikipedia mode</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>txtResult</tabstop>
  <tabstop>btnOpen</tabstop>
  <tabstop>btnPaste</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnCopy</tabstop>
 </tabstops>
 <resources>
  <include location="azc_res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
